tests/v4l2decode.cpp

```diff
@@ -715,7 +871,17 @@ int main(int argc, char** argv)
         }
     }
 #endif
-
+#if __ENABLE_WAYLAND__
```

Comment:
in fact, it does same as following "#ifndef ANDROID   takeOneOutputFrame(fd, i)"